conf/kubernetes-resource-staging-server.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: extensions/v1beta1
+kind: ReplicaSet
```

Comment:
Can we use a deployment here instead? It's a superset of the ReplicaSet in terms of features, providing rolling-update and the like. Also, I think it has recently moved to apps/v1beta1.